src/process_utility.c

```diff
@@ -37,6 +37,59 @@ static ProcessUtility_hook_type prev_ProcessUtility_hook;
 
 static bool expect_chunk_modification = false;
 
+/* Macros for DDL upcalls to PL/pgSQL */
+#define process_rename_hypertable_schema(hypertable, new_schema)		\
```

Comment:
I am not sure I find such #defines necessary. But if they are, don't they belong in catalog.h?

Reply:
They do make the code cleaner where they are called, but obviously aren't strictly necessary. I don't think they fit in catalog.c anymore than the original calls that they wrap. As I see it, these macros represent functions that typically would be implemented in process_utility if they weren't upcalls to PL/pgSQL

Reply:
Ok. That make sense. Just waiting to your response to the other comment before approving